testing/README.md

```diff
@@ -31,7 +58,51 @@ test(function example() {
   assert(!equal("hello", "world"));
   assert(equal({ hello: "world" }, { hello: "world" }));
   assert(!equal({ world: "hello" }, { hello: "world" }));
-  assertEqual("world", "world");
-  assertEqual({ hello: "world" }, { hello: "world" });
+  assert.equal("world", "world");
+  assert.equal({ hello: "world" }, { hello: "world" });
+});
+```
+
+Using `assert.strictEqual()`:
+
+```ts
+test(function isStrictlyEqual() {
+  const a = {};
+  const b = a;
+  assert.strictEqual(a, b);
+});
+
+// This test fails
+test(function isNotStrictlyEqual() {
+  const a = {};
+  const b = {};
+  assert.strictEqual(a, b);
+});
+```
+
+Using `assert.throws()`:
+
+```ts
+test(function doesThrow() {
+  assert.throws(() => {
+    throw new TypeError("hello world!");
+  });
+  assert.throws(() => {
+    throw new TypeError("hello world!");
+  }, TypeError);
+  assert.throws(
+    () => {
+      throw new TypeError("hello world!");
+    },
+    TypeError,
+    "hello"
+  );
```

Comment:
Add a counter that gets incremented in each of these callbacks, and check its count at the bottom.